### PR TITLE
Net45 update

### PR DESCRIPTION
**Pull request by khellang**

### CODE REVIEW

File: src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+using Nancy.Configuration;
```
Comment (thecodejunkie) [inline src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs]: Dude

Comment (khellang) [inline src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs]: Why do we have to use all other conventions than what's out of the box? Geez. And no .DotSettings :cry:

Comment (khellang) [inline src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs]: Dude, I'm serious. This is the only project in the .NET space I've contributed to that has these conventions. If you want me to get things right the first time, the tooling has to help me, i.e. using a .DotSettings :smile:

Comment (thecodejunkie) [inline src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs]: It's not like it's something we recently introduced =)

And yes, we should get a, proper, `.dotsettings` file committed

Comment (khellang) [inline src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs]: No, but I normally don't contribute to these satellite repos, and they don't have one. That's the only reason I get things right for Nancy core. Life's too short to manually import (and think about) namespaces.

**Pull request merged**